addons/l10n_sa_edi/views/report_invoice.xml

```diff
@@ -7,21 +7,22 @@
             <!--    Add Currency Exchange rate if different currency than SAR    -->
             <xpath expr="//div[hasclass('clearfix')]" position="after">
                 <table t-if="o.company_id.country_id.code == 'SA' and o.currency_id != o.company_id.currency_id"
-                     id="sar_amounts" t-att-style="'ltr' if lang != 'ar_001' else 'rtl'" class="row clearfix ms-auto my-3 table table-sm table-borderless">
+                    id="sar_amounts" style="page-break-inside: avoid;" class="clearfix mx-auto my-3 table-sm table-borderless">
```

Comment:
Does bootstrap have a class for that ?

Reply:
I tried finding a class for it but couldn't find one.